### ISSUE TITLE
How do I convert a pretrained Mask-RCNN model in TF1 to TF2-Keras ?

### ISSUE THREAD

**Issue by niharnsheth** [tensorflow/tpu#1001]
I am trying to take the pre-trained Mask RCNN from the Fashionpedia project [https://github.com/tensorflow/tpu/tree/master/models/official/detection/projects/fashionpedia](url) with the given checkpoints and config files and convert it to a tf saved_model format. I can use the model for inference, but I want to save it in a more recent format. I can't seem to find the right documentation [https://www.tensorflow.org/guide/migrate/migrate_tf2](url)  or examples of this. I went through the migration form TF1 to TF2 documentation, but didn't really see any methods there. Any help will be really helpful.

**Comment by cannacak** [comment]
Hey, did you overcome it ? If yes, is it possible to share with me ?
